src/QuickMarcEditor/QuickMarcCreateWrapper.js

```diff
@@ -52,14 +52,29 @@ const QuickMarcCreateWrapper = ({
 
   const onSubmit = useCallback(async (formValues) => {
     const formValuesToSave = removeDeletedRecords(formValues);
-    const controlFieldErrorMessage = checkControlFieldLength(formValuesToSave);
+    const controlFieldErrorMessage = checkDuplicateFields(
+      formValuesToSave.records,
+      '001',
+      'ui-quick-marc.record.error.controlField.multiple',
+    );
+    const duplicate010FieldErrorMessage = checkDuplicateFields(
```

Comment:
Asked Khalilah if we also need to validate it when deriving Bib or creating Holdings records

Reply:
> Asked Khalilah if we also need to validate it when deriving Bib or creating Holdings records

Ok, thank you. Please let me know once you hear from her